entrypoint.sh

```diff
@@ -17,8 +17,8 @@ oxygenctl --version
 # Temporarily ignoring that successful deploys result in 502s, that's why we || true
```

Comment:
I'm not 100% sure, but I think there may need to be a `export OXYGEN_DEPLOYMENT_ID="$INPUT_DEPLOYMENT_ID"` above the call to `oxygenctl deploy` so that the deployment ID gets sent alongside the deployment files to DMS.

Reply:
(Went ahead and added it, will likely fail the automation)